perl6.html.markdown

```diff
@@ -56,28 +56,32 @@ my @array = 1, 2, 3;
 
 say @array[2]; # Array indices start at 0 -- This is the third element
 
-say "Interpolate an array using [] : @array[]";
-#=> Interpolate an array using [] : 1 2 3
+say "Access an element of an array using [] : @array[]";
```

Comment:
No, that's wrong. I do mean interpolate. You can use "Interpolate the whole array" instead, though.